### ISSUE TITLE
BBCode not always working on new website

### ISSUE THREAD

**Issue by HoutarouOreki** [ppy/osu-web#268]
![687474703a2f2f692e696d6775722e636f6d2f65337876395a342e706e67](https://cloud.githubusercontent.com/assets/16261733/11757722/063c13e6-a064-11e5-8892-8f7b527eef19.png)

Things like [notice] and [color] work most of the time, but not always when posting from new website.

**Comment by nanaya** [comment]
mind posting the raw bbcode?

**Comment by HoutarouOreki** [comment]
`[notice][box=Setup][list][*]Keyboard: TITANUM RANGER (TK105) [Membrane]
[*]Mouse: Tracer Recon[/list][/box]
[box=Settings][list][*]DPI: [color=#FFBF00]1500[/color]
[*]Polling Rate: [color=#FF8040]1000Hz[/color]
[*]In-game Sensitivity: [color=#0080FF]0.8x
[/color][*]Raw input: [color=#00BF00]ON[/color]
[*]Resolution: [color=#00BF00]1360x768 Fullscreen[/color][/list][/box][/notice]`

**Comment by nanaya** [comment]
what's the full message? I couldn't reproduce it with just that :S (both from old and new site)

**Comment by HoutarouOreki** [comment]
I'm new here, how do I make this look like it should? ;-;

` [centre][img]https://i.ppy.sh/556ca5e9de664bec8fb6c2c91feb5f66f7146c58/687474703a2f2f7075752e73682f6a446c587a2f303862323838393261332e706e67[/img][/centre]

[notice][box=Links][list][*][url=https://www.youtube.com/channel/UCbZZQBHjMYJARCM42j4bIMQ]YouTube[/url]
[*][url=http://myanimelist.net/animelist/HoutarOsu]MyAnimeList[/url][/list][/box][/notice]
[notice][box=Setup][list][*]Keyboard: TITANUM RANGER (TK105) [Membrane]
[*]Mouse: Tracer Recon[/list][/box]
[box=Settings][list][*]DPI: [color=#FFBF00]1500[/color]
[*]Polling Rate: [color=#FF8040]1000Hz[/color]
[*]In-game Sensitivity: [color=#0080FF]0.8x
[/color][*]Raw input: [color=#00BF00]ON[/color]
[*]Resolution: [color=#00BF00]1360x768 Fullscreen[/color][/list][/box][/notice]
[notice][box=Respected Players][list][*][url=https://osu.ppy.sh/u/funorange]FunOrange[/url]
[*][url=https://osu.ppy.sh/u/rrtyui]rrtyui[/url]
[*][url=https://osu.ppy.sh/u/angelsim]Angelsim[/url]
[*][url=https://osu.ppy.sh/u/948713]- Hakurei Reimu-[/url]
[*][url=https://osu.ppy.sh/u/652457]_index[/url][/list][/box]
[box=Favorite Mappers][list][*][url=https://osu.ppy.sh/u/rlc]RLC[/url]
[*][url=https://osu.ppy.sh/u/53378]ktgster[/url](good stream maps)
[*][url=https://osu.ppy.sh/u/natsu]Natsu[/url][/list][/box][/notice]
[notice][box=Goals][color=#0080FF]2000pp - 28/03/2015
150pp score - 20/07/2015
Make an userpage - 06/08/2015
3000pp - 24.08.2015 20:40:12[/color]
[color=#BFBF00]Rank 10k - incoming
#500 in Poland - also incoming
Two digits global rank[/color]
[color=#FF0000]One digit global rank[/color][/box]
[box=][centre][url=http://info.flagcounter.com/sTkw][img]http://s01.flagcounter.com/mini/sTkw/bg_FFFFFF/txt_000000/border_FFFFFF/flags_0/[/img][/url][/centre][/box][/notice] `

You can also try this:

`[color=#0000FF][b]NM/M4M:NM
Map Link: https://osu.ppy.sh/s/372341
Title + Artist: Kaidos - Hidden Forest
Genre: Why would I know? Probably electro[/b][/color]`

![color still not working](https://cloud.githubusercontent.com/assets/16261733/11761504/c581eea4-a0c5-11e5-9497-271dc1ae51f2.png)

**Comment by nanaya** [comment]
start and end with <code>```</code>

    ```
    write [i]stuff[/i] here

    and some more.
    ```

**Comment by nanaya** [comment]
also, if the problem only happens when `color` and `b` are used across multiple lines, well, it's currently not supported :(

**Comment by HoutarouOreki** [comment]
Oh, so that's the thing. But at least there's a way around :v
```
[color=#0000FF][b]NM/M4M:NM[/b][/color]
[color=#0000FF][b]Map Link: https://osu.ppy.sh/s/372341[/b][/color]
[color=#0000FF][b]Title + Artist: Kaidos - Hidden Forest[/b][/color]
[color=#0000FF][b]Genre: Why would I know? Probably electro[/b][/color]
```

**Comment by peppy** [comment]
We should probably look into this at some point since it seems to still be affecting people.

**Comment by HoutarouOreki** [comment]
For me it looks like there is a Regular Expression or something of this kind, and you just need to replace `.*` with `(.|\n)*` between the tags. I guess I'll try looking into this, though I've never done anything like this.

**Comment by HoutarouOreki** [comment]
Closed an issue I myself opened 2.5 years ago ✌️ Feeling somehow proud of myself
